docs/WRAPPER.md

```diff
@@ -1,4 +1,4 @@
-# aragonAPI for wrappers
+# aragonAPI for Wrappers
```

Comment:
Maybe we could change to say "API for Aragon client implementations"? Nobody really knows what the "wrapper" really is 😄

Reply:
Yes, I think this is more clear